crates/gpui/src/platform/linux/dbus/mod.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::OnceLock;
+
+pub(crate) mod unity_launcher;
+
+static SESSION: OnceLock<zbus::Connection> = OnceLock::new();
+
+// TODO: this will spawn a new thread, maybe we could find a way to reuse the ashpd connection?
```

Comment:
I've looked at the `ashpd` code and it seems like there's no way to get to it, nor to pass a connection in?

https://github.com/bilelmoussaoui/ashpd/blob/f6f70c340ebea42e3ea56e539a1c9fe25d27a805/src/proxy.rs#L26-L43

Reply:
I considered making the `connection()` function in `ashpd` pub so we could use it, but I doubt they'd like making that public. The cost of an additional thread should be negligible anyway.